### ISSUE TITLE
GU0082 should warn for identical enums.

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Analyzers#160]
```cs
namespace RoslynSandbox
{
    using System;
    using NUnit.Framework;

    internal class Foo
    {
        [TestCase("1", StringComparison.Ordinal)]
        [TestCase("1", StringComparison.Ordinal)]
        public void Test(string text, StringComparison stringComparison)
        {
        }
    }
}
```
